app/assets/stylesheets/tasks.scss

```diff
@@ -0,0 +1,144 @@
+// Place all the styles related to the tasks controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+// body {
+//   background-color: lightblue;
+//   display:flex;
+// }
+//
+// div {
+//   border: 2px black solid;
+//   justify-content: space-between;
+//   display: row;
+//   // padding: 20px;
+// }
+
```

Comment:
It's a good habit to remove unused code that's commented out to clean up your files - if you need to look back at it later it'll be in your commit history :)